ci/poll-dev-extension-availability.sh

```diff
@@ -26,6 +26,9 @@ done
 attempts=1
 max=25
 
+# Installed by `TfxInstaller` in the main pipeline.
+tfx=/opt/hostedtoolcache/tfx/0.12.0/x64/bin/tfx
```

Comment:
When and how will this be updated?

Reply:
We can update this anytime by changing this line:

https://github.com/DataDog/datadog-ci-azure-devops/blob/26080d5be4b11e779937edb605c33368b422582a/.azure-pipelines/main-pipeline.yml#L153